pack/tic/def_encounter.json

```diff
@@ -0,0 +1,598 @@
+		"traits": "Monster. Elite.",
+		"type_code": "enemy"
+	},
+	{
+		"back_text": "Record in your Campaign Log <i>the tide has grown stronger</i>.\n<b>(→R1)</b>",
+		"back_flavor": "\"We've gotta pull back!\" Mr. Moore shouts over the roar of the waves. \"The sea's too rough, and the tide's gettin' worse by the minute! Stay out here any longer an' we'll capsize fer sure!\"\nYou protest, but the captain is right. The weather has gotten so bad Mr. Moore can barely control the ship. Worse yet, in just a matter of hours the tide has begun to swallow the coastal islands, leaving little for you to investigate. You nod to the captain, frustrated. This will have to do for now.",
+		"back_name": "Triumph of the Waves",
+		"code": "07166",
+		"doom": 9,
+		"double_sided": true,
+		"encounter_code": "devil_reef",
+		"encounter_position": 4,
+		"faction_code": "mythos",
+		"flavor": "Now you know why none dare venture near the black reefs. It's not just the strange sightings on the islands - there is something far older and more horrifying that dwells under the waves...",
+		"illustrator": "Mark Molnar",
+		"name": "The Devil of the Depths",
+		"pack_code": "def",
+		"position": 166,
+		"quantity": 1,
+		"stage": 2,
+		"text": "As an additional cost for you to move from one [[Ocean]] location to another while not in a vehicle, you must spend 2 actions.\n<b>Forced</b> - When your turn begins, if you are at a fully flooded location, you struggle for air: When your turn ends, if you did not enter an unflooded location, partially flooded location, or a vehicle during your turn, take 5 direct damage.",
+		"type_code": "agenda"
+	},
+	{
+		"back_text": "<b>(→R1)</b>",
+		"back_flavor": "Confident that you've found enough evidence of the Order's activities, you head back to the coast. You'd much rather continue your investigation on dry land.",
+		"back_name": "Secrets Uncovered",
+		"clues": null,
+		"code": "07167",
+		"double_sided": true,
+		"encounter_code": "devil_reef",
+		"encounter_position": 5,
+		"faction_code": "mythos",
+		"flavor": "Somewhere in these accursed reefs, the Esoteric Order of Dagon is hiding something big... but what?",
+		"illustrator": "Alan Dillingham",
+		"name": "Reef of Mysteries",
+		"pack_code": "def",
+		"position": 167,
+		"quantity": 1,
+		"stage": 1,
+		"text": "Keys can only be claimed from [[Y'ha-nthlei]] locations using the [free] ability on those locations.\n<b>Objective</b> - Find and claim as many of the following keys as you can from [[Y'ha-nthlei]] locations (purple, white, and black). If you control all 3 of these keys, advance.",
+		"type_code": "act"
+	},
+	{
+		"back_text": "Churning Waters enters play fully flooded <i>(even while it is unrevealed)</i>.",
+		"back_flavor": "\"And far out at sea, despite a high tide, I glimpsed a long, black line scarcely rising above the water yet carrying a suggestion of odd latent malignancy.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
```

Comment:
Please remove the linebreak where the quotation ends (\n- will create a bullet, so just leave - and no linebreak).